apps/win32/wdio.conf.js

```diff
@@ -1,6 +1,5 @@
 const path = require('path');
 const fs = require('fs');
-const rimraf = require('rimraf');
```

Comment:
Do we not need rimraf at all anymore?

Reply:
Seems like it's used to run a command on package.json